.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - stable
+  - "4"
+  - "0.10"
+script: npm run static
```

Comment:
imo you should define this in package.json "test": "npm run static" (for now since you will have to add more for autodeploy, see http://moox.io/statinamic/docs/usage/gh-pages/)